jf_agent/git/bitbucket_server.py

```diff
@@ -231,7 +231,7 @@ def _normalize_commit(commit, repo, branch_name, strip_text_content, redact_name
     return {
         'hash': commit['id'],
         'commit_date': datetime_from_bitbucket_server_timestamp(commit['committerTimestamp']),
-        'author': commit['author'],
+        'author': _normalize_user(commit['author']),
```

Comment:
Does it make sense to have that function check both fields instead of just `emailAddress`? Maybe not necessarily as something that gets used instantly, but to bullet-proof it a little bit in the future?

Reply:
https://github.com/Jellyfish-AI/jf_agent/pull/197/commits/8a47e489389ee22dd28113bba0ab36dcb574d522